depends/patches/python3/0010-cross-darwin-feature.patch

```diff
@@ -0,0 +1,120 @@
+diff -urN a-3.4.x/configure.ac b-3.4.x/configure.ac
```

Comment:
Okay. Just looked it over. I assume it works! Nothing jumped out at me as looking blatantly wrong.